packages/storybook/config/manager-head.html

```diff
@@ -1,7 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-  <link rel="shortcut icon" href="favicon.ico" />
-  <link rel="icon" type="image/png" href="favicon.png" sizes="192x192" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+  <link rel="manifest" href="/site.webmanifest">
```

Comment:
For unicorn deployments the icons are not shown, becaue of the `/` in the `href` attribute.